src/emulator/registry.ts

```diff
@@ -51,6 +51,9 @@ export class EmulatorRegistry {
       // Functions is next since it has side effects and
       // dependencies across all the others
       functions: 1,
+      // The Extensions emulator runs on the same process as the Functions emulator
+      // so this is a no-op.
+      extensions: 1.1,
```

Comment:
should we put extensions above functions since it has dep on functions? (I supposed the reason why we want it here even it's a no-op is to be more future-proof?)

Reply:
The type system actually forces us to include this here - I get compilation errors otherwise. Good point on moving it above tho, that would probably be more future proof